src/components/Settings/Settings.tsx

```diff
@@ -14,16 +14,18 @@ interface ModalProps {
 }
 
 enum SettingsTab {
-  General = 'general',
-  LLM = 'llm',
-  EmbeddingModel = 'embeddingModel',
-  TextGeneration = 'textGeneration',
-  ANALYTICS = 'analytics',
+  GeneralSettingsTab = 'generalSettings',
+  LLMSettingsTab = 'llmSettings',
+  EmbeddingModelTab = 'embeddingModel',
+  TextGenerationTab = 'textGeneration',
+  // RAG = "RAG",
+  ANALYTICSTab = 'analytics',
+  ChunkSizeTab = 'chunkSize',
```

Comment:
Why introduce new tabs that don't exist? Also nit but why is ```ANALYTICSTab``` all capitalised?

Reply:
> Why introduce new tabs that don't exist?

What do you mean?

> Also nit but why is ANALYTICSTab all capitalised?

Fixed